conf/modules.config

```diff
@@ -19,7 +19,10 @@ process {
     ]
 
     withName: BCFTOOLS_VIEW {
-        ext.args = ' -f PASS --output-type z'
+        ext.args = { [ "",
```

Comment:
why an empty argument? Is that needed? Have never seen that

Reply:
probably not, tbh I copied that from below (VCF2MAF)